org.adempiere.base/src/org/compiere/model/MLookupFactory.java

```diff
@@ -599,25 +599,25 @@ static private MLookupInfo getLookup_Table (Properties ctx, Language language,
 		retValue.DisplayColumn = lookupDisplayColumn;		
 		retValue.InfoWindowId = infoWindowId;
 		retValue.QueryDirect = MRole.getDefault().addAccessSQL(directQuery, TableName, true, false);
-		List<String> lookupDisplayColumns = new ArrayList<String>();
-		if (isValueDisplayed)
-			lookupDisplayColumns.add("Value");
-		lookupDisplayColumns.add(lookupDisplayColumn != null ? lookupDisplayColumn : DisplayColumn);
-		retValue.lookupDisplayColumns = lookupDisplayColumns;
-		if(list != null) {
+		List<String> lookupDisplayColumns = new ArrayList<String>();
+		if (isValueDisplayed)
+			lookupDisplayColumns.add("Value");
+		lookupDisplayColumns.add(lookupDisplayColumn != null ? lookupDisplayColumn : DisplayColumn);
+		retValue.lookupDisplayColumns = lookupDisplayColumns;
+		if(list != null) {
 			retValue.lookupDisplayColumnNames = new ArrayList<>();
 			if (isValueDisplayed)
-				retValue.lookupDisplayColumnNames.add("Value");
-			for (LookupDisplayColumn ldc : list) {
-				retValue.lookupDisplayColumnNames.add(ldc.ColumnName);
-			}
+				retValue.lookupDisplayColumnNames.add("Value");
+			for (LookupDisplayColumn ldc : list) {
+				retValue.lookupDisplayColumnNames.add(ldc.ColumnName);
+			}
 		} else {
 			retValue.lookupDisplayColumnNames = new ArrayList<>(retValue.lookupDisplayColumns); 
-		}
-		s_cacheRefTable.put(key.toString(), retValue.cloneIt());
-		return retValue;
-	}	//	getLookup_Table
-
+		}
+		s_cacheRefTable.put(key.toString(), retValue.cloneIt());
+		return retValue;
+	}	//	getLookup_Table
+

```

Comment:
please remove all the white space only changes above.

Reply:
Your new commit doesn't fix this. If you are using Eclipse, Turn off "Ignore white space" in Preferences > General > Compare/Patch.

Reply:
Also, if you have install the any edit plugin, turn off "Remove trailing whitespace" in Preferences > General > Editors > AnyEdit Tools